ci/kind/kind-setup.sh

```diff
@@ -242,6 +244,32 @@ function configure_extra_networks {
   done
 }
 
+function configure_kind_ipam_vlan_subnets {
+  if [[ -z $IPAM_VLAN_SUBNETS || -z $IPAM_VLAN_IDS ]]; then
+    return
+  fi
+  
+  IFS=',' read -r -a ipam_vlan_subnets <<< "$IPAM_VLAN_SUBNETS"
+  IFS=',' read -r -a ipam_vlan_ids <<< "$IPAM_VLAN_IDS"
+  i=0
+
+  for s in "${ipam_vlan_subnets[@]}"; do
```

Comment:
instead of s better to use meaningful name like for subnets in "${ipam_vlan_subnets[@]}"